lib/api/controllers/adminController.js

```diff
@@ -52,6 +56,37 @@ class AdminController extends BaseController {
     ]);
   }
 
+  decryptSecrets (request) {
```

Comment:
You can use async function to ensure the method will reject if `vault.decryptObject()` throw because `KUZZLE_VAULT_KEY` is not set

Reply:
It already rejects if KUZZLE_VAULT_KEY is not set, why do I need it to be async?

Reply:
I thought that controller methods must reject a promise but if you throw an error it will be catched by the funnel so it works the same